src/docs/reference/support-policy.md

```diff
@@ -124,7 +124,8 @@ please see [this document](https://github.com/ionic-team/stencil/blob/main/docs/
 
 #### Puppeteer
 
-| Stencil Version | Puppeteer v5 | Puppeteer v6 | Puppeteer v7 | Puppeteer v8 | Puppeteer v9 | Puppeteer v10 |
-|:---------------:|:------------:|:------------:|:------------:|:------------:|:------------:|:-------------:|
-|       V2        |   &#9989;    |   &#9989;    |   &#9989;    |   &#9989;    |   &#9989;    |    &#9989;    |
-|       V1        |   &#9989;    |   &#9989;    |   &#9989;    |   &#9989;    |   &#9989;    |   &#10060;    |
\ No newline at end of file
+| Stencil Version | Puppeteer v5-9 | Puppeteer v10 | Puppeteer v11 | Puppeteer v12 | Puppeteer v13 |
+|:---------------:|:--------------:|:-------------:|:-------------:|:-------------:|:-------------:|
+|       V3        |   &#10060;     |   &#9989;     |   &#9989;     |   &#9989;     |   &#9989;     |
+|       V2        |   &#9989;      |   &#9989;     |   &#9989;     |   &#9989;     |   &#9989;     |
```

Comment:
ATM, Stencil v2 doesn't support Puppeteer v11 thru v13. I don't think it'd be a huge lift to do so (and I'm open to doing it for Stencil v2), but I think at the moment we should switch some of there ✔️'s back to ❌'s:  
```suggestion
|       V2        |   &#9989;      |   &#9989;     |   &#10060;    |   &#10060;    |   &#10060;    |
```